api/lib/domain/read-models/JuryCertificationSummary.js

```diff
@@ -50,6 +63,62 @@ class JuryCertificationSummary {
   hasCompletedAssessment() {
     return this.status !== JuryCertificationSummary.statuses.STARTED;
   }
+
+  getCleaCertificationStatus() {
+    const cleaPartnerCertification = this.partnerCertifications.find(({ partnerKey }) =>
+      [PIX_EMPLOI_CLEA, PIX_EMPLOI_CLEA_V2].includes(partnerKey)
+    );
+    return this._getStatusFromPartnerCertification(cleaPartnerCertification);
+  }
+
+  getPixPlusDroitMaitreCertificationStatus() {
+    const pixPlusDroitMaitrePartnerCertification = this.partnerCertifications.find(
+      ({ partnerKey }) => partnerKey === PIX_DROIT_MAITRE_CERTIF
+    );
+    return this._getStatusFromPartnerCertification(pixPlusDroitMaitrePartnerCertification);
+  }
+
+  getPixPlusDroitExpertCertificationStatus() {
+    const pixPlusDroitExpertPartnerCertification = this.partnerCertifications.find(
+      ({ partnerKey }) => partnerKey === PIX_DROIT_EXPERT_CERTIF
+    );
+    return this._getStatusFromPartnerCertification(pixPlusDroitExpertPartnerCertification);
+  }
+
+  getPixPlusEduAutonomeCertificationStatus() {
+    const pixPlusEduAutonomePartnerCertification = this.partnerCertifications.find(
+      ({ partnerKey }) => partnerKey === PIX_EDU_FORMATION_INITIALE_2ND_DEGRE_AUTONOME
+    );
+    return this._getStatusFromPartnerCertification(pixPlusEduAutonomePartnerCertification);
+  }
+
+  getPixPlusEduAvanceCertificationStatus() {
+    const pixPlusEduAvancePartnerCertification = this.partnerCertifications.find(({ partnerKey }) =>
+      [PIX_EDU_FORMATION_INITIALE_2ND_DEGRE_AVANCE, PIX_EDU_FORMATION_CONTINUE_2ND_DEGRE_AVANCE].includes(partnerKey)
+    );
+    return this._getStatusFromPartnerCertification(pixPlusEduAvancePartnerCertification);
+  }
+
+  getPixPlusEduExpertCertificationStatus() {
+    const pixPlusEduExpertPartnerCertification = this.partnerCertifications.find(
+      ({ partnerKey }) => partnerKey === PIX_EDU_FORMATION_CONTINUE_2ND_DEGRE_EXPERT
+    );
+    return this._getStatusFromPartnerCertification(pixPlusEduExpertPartnerCertification);
+  }
+
+  getPixPlusEduFormateurCertificationStatus() {
+    const pixPlusEduFormateurPartnerCertification = this.partnerCertifications.find(
+      ({ partnerKey }) => partnerKey === PIX_EDU_FORMATION_CONTINUE_2ND_DEGRE_FORMATEUR
+    );
+    return this._getStatusFromPartnerCertification(pixPlusEduFormateurPartnerCertification);
+  }
+
+  _getStatusFromPartnerCertification(partnerCertification) {
+    if (!partnerCertification) {
+      return partnerCertificationStatus.NOT_TAKEN;
+    }
+    return partnerCertification.acquired ? partnerCertificationStatus.ACQUIRED : partnerCertificationStatus.REJECTED;
+  }
```

Comment:
On pourrais aller plus loin dans la factorisation
```suggestion
  _getStatusFromPartnerCertification(partnerCertificationKeys) {
    const partnerCertification = this.partnerCertifications.find(
      ({ partnerKey }) => partnerCertificationKeys.includes(partnerKey)
    );
    if (!partnerCertification) {
      return partnerCertificationStatus.NOT_TAKEN;
    }
    return partnerCertification.acquired ? partnerCertificationStatus.ACQUIRED : partnerCertificationStatus.REJECTED;
  }
```

Reply:
Done